app/controllers/dashboards_controller.rb

```diff
@@ -8,6 +8,9 @@ class DashboardsController < ApplicationController
 
   def show
     authorize! :dashboard
-    @collections = authorized_scope(Collection.all)
+    @presenter = DashboardPresenter.new(
+      collections: authorized_scope(Collection.all),
+      drafts: Work.where(state: 'first_draft')
```

Comment:
You can also use `Work.with_state(:first_draft)` here, I believe. our state machine library generates these scopes for us. Not a big deal either way.